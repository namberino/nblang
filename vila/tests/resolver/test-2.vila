mut test = "test";
{
    mut test = test; // error
}
