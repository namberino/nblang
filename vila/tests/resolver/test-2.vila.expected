On line: 3, Error at 'test': Can't read local variable in its initializer
